brightchain-lib/src/lib/access/tuple.ts

```
@@ -0,0 +1,57 @@
+    primeWhitened: WhitenedBlock,
+    whiteners: WhitenedBlock[],
+    randomBlocks: RandomBlock[],
+  ): Promise<EphemeralBlock> {
+    return this.getService().xorDestPrimeWhitenedToOwned(
+      creator,
+      primeWhitened,
+      whiteners,
+      randomBlocks,
+    );
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)